ResearchKit/Consent/ORKConsentSignatureFormatter.m

```diff
@@ -58,7 +58,11 @@ - (NSString *)HTMLForSignature:(ORKConsentSignature *)signature {
             if (signature.familyName) {
                 [names addObject:signature.familyName];
             }
-            nameStr = [names componentsJoinedByString:@"&nbsp;"];
+            if ([ORKConsentSignature currentLocalePresentsFamilyNameFirst]) {
+                nameStr = @"";
```

Comment:
A family name + given name without a space is very common in the Asian country because it's not mandatory. However, maybe you are right, I think it could be more formal with a space in an official document.